pkg/kubernetes/client.go

```diff
@@ -92,6 +102,40 @@ func (c *Client) GetLoadTest(ctx context.Context, loadTest string) (*apisLoadTes
 	return result, nil
 }
 
+// ListLoadTest returns list of load tests.
+func (c *Client) ListLoadTest(ctx context.Context, opt ListOptions) ([]apisLoadTestV1.LoadTest, error) {
+	k8sOpt := metaV1.ListOptions{}
```

Comment:
I think we should introduce pagination here to avoid a possibly overloading the service if there are a lot of load tests.
See Limit and Continue under [ListOptions](https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#ListOptions)